doc/api/esm.md

```diff
@@ -573,7 +573,7 @@ The resolve hook returns the resolved file URL and module format for a
 given module specifier and parent file URL:
 
 ```js
-const baseURL = new URL(`${process.cwd()}/`, 'file://');
+const baseURL = new URL(`${process.cwd()}`, 'file://');
```

Comment:
Not sure about this change, but if we're going to do it, there's no need to use a template string anymore:

```suggestion
const baseURL = new URL(process.cwd(), 'file://');
```

Reply:
doc: removed template string